README_CONTAINER_IMAGE.md

```diff
@@ -47,3 +47,26 @@ Here is a detailed explanation of the options used in the command above:
 Further usage examples are available in the [examples directory](examples/) with samples of how to use the image from within OpenShift.
 
 Additional usage information for images built from `playbook2image` like this one can be found in the [playbook2image examples](https://github.com/aweiteka/playbook2image/tree/master/examples).
+
+## Running Ansible as a System Container
+
+Building the System Container: See the [BUILD.md](BUILD.md).
+
+Create `/etc/ansible` directory on the host machine and copy inventory file (hosts) into it.
+
+Copy ssh public key of the host machine to master and nodes machines in the cluster.
+
+If the inventory file needs additional files then it can use the path `/var/lib/openshift-installer` in the container as it is bind mounted from the host (controllable with `VAR_LIB_OPENSHIFT_INSTALLER`).
```

Comment:
Re-reading this I understood it better, I think. Though I'm curious as to what is the corresponding path in the host?